source/MRMesh/MRPolylineComponents.cpp

```diff
@@ -1,7 +1,38 @@
 #include "MRPolylineComponents.h"
+#include "MRPolyline.h"
 #include "MRPolylineTopology.h"
 #include "MRPolylineEdgeIterator.h"
 #include "MRTimer.h"
+#include "MRVector2.h"
+#include "MRVector3.h"
+
+namespace
+{
+
+using namespace MR;
+
+using UndirectedEdgeMap = Vector<UndirectedEdgeId, UndirectedEdgeId>;
```

Comment:
this can be put in MRMeshFwd.h if you like

Reply:
Moved the declaration.